core/translate.c

```diff
@@ -1438,7 +1438,7 @@ record_translation_info(dcontext_t *dcontext, fragment_t *f, instrlist_t *existi
      * final entry for the inserted jmp, have recreate_ know about it and cut
      * in half the typical storage reqts.
      */
-#   define NUM_INITIAL_TRANSLATIONS 2
```

Comment:
I am not sure about that one. While I think aligning the defines with the function code looks quite pleasing, I also think it might be slightly misleading, as visually it may suggest that the macro has a function scope.

also, I only found a few instances using that style, so I think adding this to the rules will cause a lot of errors.

If we still want to add a rule, we could just keep track of opening and closing brackets '{ }' to detect if we are inside a function body, that should be good enough for the majority of the cases (in C code anyways)